vendor/modules.txt

```diff
@@ -0,0 +1,32 @@
+# github.com/bitrise-community/steps-cordova-archive v0.0.0-20181212111956-3095639b003c
+## explicit
+github.com/bitrise-community/steps-cordova-archive/cordova
+# github.com/bitrise-community/steps-ionic-archive v0.0.0-20181207135302-7869644a568b
+## explicit
+github.com/bitrise-community/steps-ionic-archive/jsdependency
```

Comment:
Can you please check and replace bitrise-community references with bitrise-io; then run a go module update.
bitrise-tools should disappear too if you do that.